book/src/game-data.md

```diff
@@ -81,7 +81,8 @@ impl<'a, 'b> CustomGameDataBuilder<'a, 'b> {
 
 impl<'a, 'b> DataInit<CustomGameData<'a, 'b>> for CustomGameDataBuilder<'a, 'b> {
     fn build(self, world: &mut World) -> CustomGameData<'a, 'b> {
-        let pool = world.read_resource::<ThreadPool>().clone();
+        // Get a handle to the `ThreadPool`.
+        let pool = world.read_resource::<Arc<ThreadPool>>().clone();
```

Comment:
Why did we change to `Arc`?

Reply:
because we use a Arc<ThreadPool> and the example was wrong.